src/backend/impl_safe/cursor.rs

```diff
@@ -30,7 +30,7 @@ impl<'env> BackendRoCursor<'env> for RoCursorImpl<'env> {
     {
         // FIXME: Don't allocate.
         let key = key.as_ref().to_vec();
-        IterImpl(Box::new(self.0.iter().skip_while(move |&(k, _)| k != &key[..])))
+        IterImpl(Box::new(self.0.iter().skip_while(move |&(k, _)| k < key.as_slice())))
```

Comment:
This was making clippy complain unnecessarily. Rather than silencing it with a pragma, using `as_slice` made it shut up.